src/app/global-header/global-header.component.html

```diff
@@ -1,33 +1,59 @@
+              />
+            </defs>
+          </svg>
+        </div>
+        <div class="col-9 col-md-10 text-left px-0">
+          <h5 class="card-title fs-14 font-600 br-l ml-2 pl-3">
+            <div class="text-gray-0 fs-14">DEPARTMENT OF</div>
+            <div class="fs-22 heading3">Basic Education</div>
 
-                        <div class="text-orange fs-12">GOVERNMENT OF UTTAR PRADESH</div>
-                    </h5>
-                </div>
-            </div>
+            <div class="text-orange fs-12">GOVERNMENT OF UTTAR PRADESH</div>
+          </h5>
         </div>
+      </div>
     </div>
-    <div class="heading2 pb-10 mt-4" *ngIf="showTitle">
-        AVSAR Learner Wallet
-    </div>
-    <div class="pb-20" *ngIf="showTitle">
-        {{'WELCOME_TO_YOUR_ACADEMIC_WALLET' | translate}}
-    </div>
-</div>
\ No newline at end of file
+  </div>
+  <div class="heading2 pb-10 mt-4" *ngIf="showTitle">AVSAR Learner Wallet</div>
+  <div class="pb-20" *ngIf="showTitle">
+    {{ "WELCOME_TO_YOUR_ACADEMIC_WALLET" | translate }}
+  </div>
+
+  <div style="position: fixed; top: 0; right: 0; padding: 10px">
+    <span (click)="changeTheme()">
```

Comment:
@apurvaubade Revert this file. We're not going to add option to switch the theme